app/src/app/[lng]/page.tsx

```diff
@@ -413,18 +414,16 @@ export default function Home({ params: { lng } }: { params: { lng: string } }) {
               color="interactive.control"
               letterSpacing="wide"
             >
-              The data you have submitted is now officially incorporated into
-              your city&apos;s {year} GHG Emissions Inventory, compiled
-              according to the GPC Basic methodology.{" "}
+              {t("gpc-data-description")} {year} {t("add-data-description-ctn")}
```

Comment:
Here using a placeholder in the translation string would be better.  
Relevant docs:  
https://www.i18next.com/translation-function/interpolation  
https://react.i18next.com/latest/trans-component#using-with-react-components